### PR TITLE
registreren functie met keycloak toegevoegd

### PR DESCRIPTION
**Pull request by ZakariaICT**

### CODE REVIEW

**Comment by pepijn03** [comment]
Reviewed merge conflict, accepted all incoming changes

**Review by pepijn03 — DISMISSED** [review]

File: .env
Comment (MilanBrock) [inline .env:1]: Add to gitignore

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_DB=mydatabase
+POSTGRES_USER=myuser
+POSTGRES_PASSWORD=mypassword
+KEYCLOAK_ADMIN=myadmin
+KEYCLOAK_ADMIN_PASSWORD=myadminpassword
```
Comment (Popodil) [inline .env]: Add to .gitignore

File: .env
Comment (Fontie) [inline .env:1]: Add to .gitignore

File: src/main/resources/application.properties
Comment (Fontie) [inline src/main/resources/application.properties:1]: add to .gitignore

File: src/main/java/com/babymonitor/identity/services/UserCreation.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+    private Keycloak getKeycloakClient() {
+        return KeycloakBuilder.builder()
+                .serverUrl(authServerUrl)
+                .realm("TaskService")
+                .clientId(clientId)
+                .username("hassan")
+                .password("hassan")
```
Comment (MilanBrock) [inline src/main/java/com/babymonitor/identity/services/UserCreation.java]: Toevoegen in env file

Comment (Popodil) [inline src/main/java/com/babymonitor/identity/services/UserCreation.java]: To ENV file

**Review by Fontie — DISMISSED** [review]
Good for now. Please remove the passwords from the repo.

File: src/main/resources/application.properties
Comment (MilanBrock) [inline src/main/resources/application.properties:1]: Env file !!

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,2 +1,7 @@
 spring.application.name=identity
-server.port=8091
\ No newline at end of file
+keycloak.auth-server-url=http://localhost:8080
+keycloak.realm=TaskService
+keycloak.client-id=taskservice-client-api
+keycloak.username=myadmin
+keycloak.password=myadminpassword
+
```
Comment (Popodil) [inline src/main/resources/application.properties]: To ENV file

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: http://localhost:8080/realms/TaskService
```
Comment (MilanBrock) [inline src/main/resources/application.yml]: envvvvvvvvvvvvvvv

Comment (Popodil) [inline src/main/resources/application.yml]: To ENV File

**Review by MilanBrock — DISMISSED** [review]

**Review by Popodil — DISMISSED** [review]

**Review by Popodil — DISMISSED** [review]

**Pull request closed without merging**